go-controller/pkg/ovn/policy.go

```diff
@@ -30,10 +33,15 @@ type networkPolicy struct {
 	podHandlerList  []*factory.Handler
 	svcHandlerList  []*factory.Handler
 	nsHandlerList   []*factory.Handler
-	localPods       map[string]*lpInfo //pods effected by this policy
-	portGroupUUID   string             //uuid for OVN port_group
-	portGroupName   string
-	deleted         bool //deleted policy
+
+	// localPods is a list of pods affected by ths policy
+	// this is a sync map so we can handle multiple pods at once
+	// map of string -> *lpInfo
+	localPods sync.Map
+
+	portGroupUUID string //uuid for OVN port_group
+	portGroupName string
+	deleted       bool //deleted policy
```

Comment:
nit and you didn't add this but all the other comments for variable types are above the variable they comment on

Reply:
Those were existing comments that just got re-whitespaced.